.github/workflows/tests.yaml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     paths-ignore: ['*.md', 'CODEOWNERS', 'LICENSE']
     branches:
-    - 'master'
+    - 'main'
     - 'release/*'
```

Comment:
Ideally the branches section would include both `master` and `main` for a period of time before cutting over.